community/examples/fluent-tutorial.yaml

```diff
@@ -0,0 +1,167 @@
+
+        destination: /tmp/install-deps.sh
+      - type: shell
+        content: |
+          #!/bin/bash
+          cat <<- "EOF" >> /etc/bashrc
+          export LD_LIBRARY_PATH=/shared/apps/fluent/v222/TurboGrid/tools/Prime/lib/:${LD_LIBRARY_PATH}
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/commonfiles/fluids/lib/linx64/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/commonfiles/fluids/lib/linx64/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/fluent/fluent22.2.0/multiport/lnamd64/mpi/shared/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/shared_files/syslib/linx64
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/aisol/lib/linx64/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/polyflow/polyflow22.2.0/lnamd64/libs/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/tp/IntelMKL/2021.3.0/linx64/lib/intel64/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/fluent/lib/lnamd64/
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/shared/apps/fluent/v222/dcs/
+          export INTELMPI_ROOT=/shared/apps/intel/compilers_and_libraries_2018.5.274/linux/mpi/intel64
+          source /shared/apps/intel/parallel_studio_xe_2018.4.057/psxevars.sh
+          EOF
+
+        destination: /tmp/install-bashrc.sh
+      - type: shell
+        content: |
+          #!/bin/bash
+          mkdir /tmp/intel
+          cd /tmp/intel
+          wget https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13741/l_mpi_2018.4.274.tgz
+          tar xvf l_mpi_2018.4.274.tgz
+          cd l_mpi_2018.4.274
+          patch silent.cfg <<- "EOF1"
+          16:07:46.199308305 +0000
+          @@ -6,13 +6,13 @@
+           # snpat       - the serial number pattern (ABCD-01234567)
+
+           # Accept EULA, valid values are: {accept, decline}
+          -ACCEPT_EULA=decline
+          +ACCEPT_EULA=accept
+
+           # Optional error behavior, valid values are: {yes, no}
+           CONTINUE_WITH_OPTIONAL_ERROR=yes
+
+           # Install location, valid values are: {/opt/intel, filepat}
+          -PSET_INSTALL_DIR=/opt/intel
+          +PSET_INSTALL_DIR=/shared/apps/intel
+
+           # Continue with overwrite of existing installation directory, valid values are: {yes, no}
+           CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
+          EOF1
+
+          sudo ./install.sh -s silent.cfg
+
+        destination: /tmp/install-mpi.sh
+
+  - source: ./modules/scripts/startup-script
```

Comment:
I missed this one, but it should also be embedded: `modules/...` rather than `./modules/...`

Reply:
Got it.